packages/rest-hooks/tests/restClient.test.ts

```diff
@@ -356,7 +356,98 @@ describe('Test Rest Client', () => {
   test.skip('Test Straight Patch', async () => {})
   test.skip('Test PatchFn', async () => {})
 
-  test.skip('Test Straight Delete', async () => {})
-  test.skip('Test DeleteFn', async () => {})
+  test('Test Straight Delete', async () => {
+    const restClient = createRestClient(
+      defaultClientConfig,
+      defaultServerConfig
+    )
+
+    const rsp = (await restClient.delete(
+      '/simple-delete/user/favorite-songs'
+    )) as unknown as StcRestTest.TestResponse
+
+    const [requestInfo, responseBody] = rsp
+
+    expect(requestInfo?.url).toEqual(
+      'http://fakehost.com:5023/simple-delete/user/favorite-songs'
+    )
+    expect(requestInfo?.method).toEqual('DELETE')
+    expect(requestInfo?.headers.authorization).toEqual(
+      'Bearer testing-access-token'
+    )
+    // Normally, you wouldn't expect a body for DELETE.
+    expect(requestInfo?.body).toBeNull()
+    expect(responseBody).toEqual({ message: 'delete succesful' })
+
+  })
+
+  test('Test DeleteFn', async () => {
+
+    const restClient = createRestClient({
+      ...defaultClientConfig,
+      getAccessToken: () => 'another-access-token'
+    }, defaultServerConfig)
+
+    // simple delete
+
+    const deleteFn = restClient.createDeleteFn('/simple-delete')
+    rsp = await deleteFn({}) as unknown as StcRestTest.TestResponse;
```

Comment:
I just had a look at DeleteFn definition in `restHooksTypes.ts`, I should have made `restParams` an optional argument.   I suggest the following
(1) for `export type DeleteFn = (restParams: RestParams) => Promise<AxiosResponse>`, make restParams optional. (2) once optional, call ion this line can migrate from `deleteFn({})` => `deleteFn()`